covid_moonshot/analysis/plots.py

```diff
@@ -252,6 +253,40 @@ def plot_convergence(
     return fig
 
 
+def plot_poor_convergence_fe_table(
+    runs: List[int], runs_details: List[int], energy_cutoff: float = 1.0,
```

Comment:
Types need to be updated here. If you aren't using it already, I highly recommend [mypy](http://mypy-lang.org/)! It can catch a lot of potential errors your code based on the type annotations you provide.

Reply:
You might use a single argument that is just a `List[Run]` here, since `Run` just combines `RunDetails` and `RunAnalysis`.

Reply:
So far I've been following the convention of suffixing energies in kcal/mol with `_kcal`, and otherwise assuming `kT`. It looks like `energy_cutoff` is supposed to be in kcal/mol here.

Reply:
Thanks for the `mypy` recommendation, it's great!